common/src/View/UVCameraTool.cpp

```diff
@@ -90,7 +90,6 @@ void UVCameraTool::mouseScroll(const InputState& inputState)
     {
       const float zoomFactor =
         1.0f + inputState.scrollZ() * factor + inputState.scrollY() / 50.0f * factor;
-      const auto mousePos = vm::vec2f{inputState.mouseX(), inputState.mouseY()};
```

Comment:
If this is a fixup for a previous commit, then please create a `fixup!` commit so that it gets squashed.

Reply:
I'm not super familiar/confident with (auto)squash workflows, I'll manually rebase and squash it into the right commit